0.5/articles/unit-testing-elements.md

```diff
@@ -421,11 +421,11 @@ As we can see, when we run `wct` once again we're still all green:
 
 **Go forth and write tests!** They can be simple and powerful.
 
-Remember, testing Web Components isn't vastly different to unit testing the JavaScript components you build everyday. Many of the same best practices apply. You're still working with events, objects and an API.
+Remember, testing Web Components isn't vastly different to unit testing the JavaScript components you build every day. Many of the same best practices apply. You're still working with events, objects and an API.
 
 The beauty of sticking with Mocha and Chai is tests can execute equally well in both the browser and continuous integration.
 
-Thanks for reading and do let us know if you have any questions about unit testing your elements!
+Thanks for reading and do let us know if you have any questions about unit testing your elements! (How/where?)
```

Comment:
Fair enough. Is there a page with that on it somewhere (I vaguely recall seeing something like that)? Hrm, perhaps this <https://www.polymer-project.org/0.5/resources/faq.html#contributing>